interface/admin.py

```diff
@@ -13,16 +13,21 @@
 
 @admin.register(Assignment)
 class AssignmentAdmin(admin.ModelAdmin):
-    actions = ['download_submissions']
+    actions = ['download_submissions', 'download_all_submissions']
 
-    def download_submissions(self, request, queryset):
+    def get_submission_set(self, request, queryset):
         if queryset.count() != 1:
             messages.error(request, 'Only one assignment can be selected')
             return
 
         assignment = queryset[0]
         submission_set = assignment.submission_set.order_by('timestamp')
 
+        return submission_set
+
+    def download_submissions(self, request, queryset):
```

Comment:
Name it better. Something like `download_review_submissions` should do it. Also don't forget to rename the part at the end of the function too.